src/main/java/at/hannibal2/skyhanni/features/garden/farming/GardenBestCropTime.kt

```diff
@@ -115,7 +115,7 @@ class GardenBestCropTime {
 
             val color = if (isCurrent) "§e" else "§7"
             val contestFormat = if (GardenNextJacobContest.isNextCrop(crop)) "§n" else ""
-            val currentTier = GardenCropMilestones.getTierForCropCount(crop.getCounter(), crop)
+            val currentTier = GardenCropMilestones.getTierForCropCount(crop.getCounter(), crop, allowOverflow = true)
```

Comment:
maybe set allowOverflow default to true?

Reply:
It seems to be tied 4/4 in true or false usage (aside from contexts where it's determined by a variable). I made it default to false because it's probably better to accidentally break the overflow milestone feature in some place than accidentally make it count overflow milestones where it shouldn't if something new like dedication gets added. But I can change the default if you still think it's better.

Reply:
default disabled is fine.